drivers/i2c/tt_stm32_i2c_byte_poll_api.c

```diff
@@ -31,6 +31,18 @@ LOG_MODULE_REGISTER(tt_stm32_i2c_api);
 
 #include "i2c-priv.h"
 
+#define LOG_ERR_RATELIMIT(ms, ...)                                                                 \
```

Comment:
Should we move this change to a separate PR?

Reply:
Could yeah, it's low-hanging fruit. There are a couple of commits that are kind of tangential to the main intent, just to help debug

Reply:
I might not even need that change outside of this PR, so will likely just remove it later.